apps/src/publicKeyCryptography/ModuloClock.jsx

```diff
@@ -105,22 +112,34 @@ const ModuloClock = React.createClass({
       ];
     } else {
       // Render distinct segments
-      const wedgePath = createWedgePath((2 * Math.PI / modulus) - segmentGapRadians);
-      return _.range(0, modulus).map(n => (
-        <path
-          key={n}
-          d={wedgePath}
-          transform={`rotate(${(n - 0.5) * 360 / modulus} 50 50)`}
-          fill={n <= result ? COLOR.fullWedge : COLOR.emptyWedge}
-        />
-      ));
+      const segmentCount = modulus;
+      const wedgePath = createWedgePath((2 * Math.PI / segmentCount) - segmentGapRadians);
+      return _.range(0, segmentCount).map(n => {
+        // When running we want R=0 to be a full pie, for a nice smooth
+        // animation (that doesn't skip a pie slice).
+        // When not running we want R=0 to be an empty pie, to accurately
+        // represent the final result.
+        // Also when not running we enable a fade effect.
+        // Together these create the special situation where you animate to a
+        // full pie (d%m=0), stop animating, and fade to an empty pie.
+        const isSegmentFull = n < result || (isRunning && 0 === result);
+        return (
+          <path
+            key={n}
+            d={wedgePath}
+            transform={`rotate(${n  * 360 / segmentCount} 50 50)`}
+            fill={isSegmentFull ? COLOR.fullWedge : COLOR.emptyWedge}
+            style={!isRunning ? {transition: 'fill 1.5s'} : {}}
+          />
+        );
```

Comment:
this logic would be a good candidate for a unit test, though, I am sympathetic to the argument that this feature is perhaps not the highest priority for test coverage given the low churn.